rmgpy/molecule/resonance.py

```diff
@@ -315,6 +315,13 @@ def _generate_resonance_structures(mol_list, method_list, keep_isomorphic=False,
         # Move to the next resonance structure
         index += 1
 
+    # check net charge
+    for mol in mol_list:
+        if mol.getNetCharge() != 0:
+            raise ResonanceError('Resonance generation gave a net charged molecule:\n{0}'
+                                 'Ions are not yet supported in RMG.'.format(
+                mol.toAdjacencyList()))
```

Comment:
You have a newline before {0}, so it seems like this would print like:
```
Resonance generation gave a net charged molecule:
smilesstringIons are not yet supported in RMG.
```

Could you double-check? Also, could you indent the last line to match the prior line?

Reply:
I thought printing an adjList automatically ends with a new line. Trying it our I get:
```
Resonance generation gave a net charged molecule:
1 S u0 p1 c0 {2,D} {3,D}
2 O u0 p2 c0 {1,D}
3 O u0 p2 c0 {1,D}
Ions are not yet supported in RMG.
```

Reply:
You're right, for some reason I thought it was including the SMILES.